account-lookup-service/values.yaml

```diff
@@ -437,7 +437,7 @@ als-oracle-pathfinder:
         repository: mojaloop/als-oracle-pathfinder
         tag: latest
         pullPolicy: Always
-        command: mysql -h test-account-lookup-mysql -P $db_port -u account_lookup --password=OdonPianoX account_lookup -e "SET @service_name='$service_name'; source /opt/als-oracle-pathfinder/init-account-lookup.sql;";
+        command: mysql -h $db_host -P $db_port -u $db_user --password=$db_password $db_database -e "SET @service_name='$service_name'; source /opt/als-oracle-pathfinder/init-account-lookup.sql;";
```

Comment:
I would not recommend this approach. The API should instead be called to load the ALS routing tables via HTTP so that it is generic, and it will provide the flexibility to for the Oracle to be hosted outside of a Switch implementation.

Also, not sure why the ALS-Oracle-Pathfinder is directly accessing the Central-Ledger DB (write/read new mno table). It should really be using its own hosted DB or unique Database.

Reply:
We can discuss and perform those in a later release @mdebarros, this PR is about fixing those hardcoded values only, the rest are already in master.

Reply:
@vbarzokas I think this is the first time we realized it (DB tables being created in central ledger, etc). Can you please ensure that follow up items to address the points that Miguel raised are put in the backlog, that can help unblock this.

Reply:
https://github.com/mojaloop/als-oracle-pathfinder/issues/15
https://github.com/mojaloop/als-oracle-pathfinder/issues/16

I take responsibility for this. Note that at the time there would've been a few Mowali-specific considerations that led to this functionality, and that those things simply haven't been addressed since donation to OSS (probably because I copied parts of the implementation, and didn't think too hard about it).

Reply:
Thanks @msk- for addressing these, lets consider these kind-of technical debt items which I'm confident you'll pursue and address soon.